packages/exhibitor/extra/dcos_zk_backup.py

```diff
@@ -41,12 +41,15 @@ def run_command(cmd: str, verbose: bool) -> None:
 
 def _is_zookeeper_running(verbose: bool) -> bool:
     """
-    Exhibitor and therefore ZooKeeper is in active state (running).
+    Returns whether the ZooKeeper process that Exhibitor controls is running.
     """
+    zk_pid_file = Path('/var/lib/dcos/exhibitor/zk.pid')
+    zk_pid = int(zk_pid_file.read_text())
     try:
-        run_command('systemctl is-active --quiet dcos-exhibitor', verbose)
+        # Check whether the ZooKeeper that Exhibitor controls is running.
+        run_command('kill -0 {zk_pid}'.format(zk_pid=zk_pid), verbose)
```

Comment:
Quote the man page here:
```
       If  signal  is  0, then no actual signal is sent, but error checking is
       still performed.
```

Reply:
(can do that later, or just in backports, or not at all)

Reply:
I added this to a backports.